### PR TITLE
Adding unit tests for decodable request wrappers

### PR DESCRIPTION
**Pull request by hamtiko**

### CODE REVIEW

**Comment by hamtiko** [comment]
@rynecheow I have added unit tests for extensions I added before based on your suggestion.

Please check and give your thoughts.

Please note I have removed `@testable import Alamofire` from test file as it give following error:
`Module 'Alamofire' was not compiled for testing`

Let me know if it should be kept.

Thanks.

**Comment by rynecheow** [comment]
@hamtiko I'll be checking the PR today. As for your import error, you should be able to get it resolved if your Carthage dependencies are built with debug configuration.

File: Tests/RxAlamofireTests/RxAlamofireTests.swift [hunk 1]
```diff
@@ -6,14 +6,18 @@ import RxCocoa
 import RxSwift
 import XCTest
 
-@testable import Alamofire
```
Comment (rynecheow) [inline Tests/RxAlamofireTests/RxAlamofireTests.swift:9]: As mentioned, this is probably able to be resolved if you install Carthage dependencies with debug config.

Comment (hamtiko) [reply Tests/RxAlamofireTests/RxAlamofireTests.swift:9]: But is it really necessary? We are not testing Alamofire target right?

Comment (rynecheow) [reply Tests/RxAlamofireTests/RxAlamofireTests.swift:9]: Ah I see - we are not utilising any internal functions in tests right now. It is alright to ignore it for now.

Comment (hamtiko) [reply Tests/RxAlamofireTests/RxAlamofireTests.swift:9]: Ok. Thanks

**Comment by hamtiko** [comment]
@rynecheow do you have any idea why the check build is failing? I have tried to re-run 2 types.

**Comment by rynecheow** [comment]
I think probably because Danger ruby are unable to run or forked commits. May need to find an alternative for it though, potentially need to disable it for forks now.

**Comment by hamtiko** [comment]
Do you think I should close this PR and create new from main repo until build workflow we will be updated?

**Comment by rynecheow** [comment]
@hamtiko Sounds like a good idea. Lets do that while we improve the build workflow.

**Comment by hamtiko** [comment]
@rynecheow I have created new one https://github.com/RxSwiftCommunity/RxAlamofire/pull/182

**Pull request closed without merging**